### PR TITLE
Fixed parentheses that are interp-ed as grouped expression

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

File: app/mailers/notifier.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ def notify(notification)
     @reply_link = reply_link(@notification)
 
     # Encrypting
-    @signed_message = verifier.generate ({ member_id: @notification.recipient.id, type: :send_notification_email })
+    @signed_message = verifier.generate(member_id: @notification.recipient.id, type: :send_notification_email)
```
Comment (maco) [inline app/mailers/notifier.rb]: was it intentional to ditch the `{}`?

Comment (Br3nda) [inline app/mailers/notifier.rb]: So, i need I'm really a python dev who pretends to know ruby.
Rubocop complained about the {} being unnecessary, hinting that ruby treats it as a hash regardless. I took it at it's word.

Comment (Br3nda) [inline app/mailers/notifier.rb]: read up some - and you're right. Good catch. Fixing this now

**Comment by maco** [comment]
It was totally possible I was also being Pythonic, but it seemed worth
checking

**Comment by Br3nda** [comment]
Could someone with the powers fix up coveralls? I would love to be able to check that CI touched the lines that changed

image

**Comment by CloCkWeRX** [comment]
There's something funny going on - its clearly there on dev via git.

I got an email notification just yesterday from data@gitshield.com - 	Tue, Dec 6, 2016 at 5:22 PM (thats before you authored the PR apparently though)
```

File Name
 
/notifier.rb

 
Pattern Matched
 
Possible Ruby secret key(s)
```

That's clearly wrong, but I'm wondering if a similar thing is happening with other apps and they think they are protecting us.

**Pull request merged**